Tasks/DownloadPackageV1/package.ts

```diff
@@ -154,13 +154,11 @@ export abstract class Package {
         return new Promise<string>(async (resolve, reject) => {
             this.getPackageMetadata(this.feedConnection, routeValues, queryParams, this.getPackagesAreaId)
             .then(packages => {
-                if(packages["id"] == packageId){
+                if(packages["id"] == packageId) {
                     let versions = packages["versions"];
                     tl.debug("Found " + versions?.length + " packages matching search pattern " + packageId);
-                    for (let i = 0; i < versions?.length; i++) {
-                        if (versions[i]["isLatest"] && versions[i]["isListed"]) {
-                            return resolve(versions[i]["normalizedVersion"]);
-                        }
+                    if (versions[0]["isListed"]) {
+                        return resolve(versions[0]["normalizedVersion"]);
```

Comment:
I am unsure if this resolves this issue as it would just return the first package from the API response. I am not sure if our API guarantees the first version is the latest. Can we confirm? 

IIRC from my testing, I think the bug this PR is trying to fix only occurs if there is no package version with "isLatest" equal to true. I was not aware this was a valid case we also might want to look into see if this is true, and if so why?

Reply:
@jiaw37 confirmed